src/main/kotlin/frc/robot/subsystems/IntakeSubsystem.kt

```diff
@@ -0,0 +1,108 @@
+package frc.robot.subsystems
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.SparkMaxAbsoluteEncoder
+import com.revrobotics.CANSparkMaxLowLevel
+import edu.wpi.first.math.controller.PIDController
+import edu.wpi.first.math.controller.ArmFeedforward
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.wpilibj.Encoder
+import edu.wpi.first.wpilibj.motorcontrol.MotorController
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+
```

Comment:
Comments everywhere. Look into KDoc comments. https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax Look into this.